services/src/main/java/org/keycloak/authorization/admin/PolicyService.java

```diff
@@ -176,6 +176,7 @@ public Response findAll(@QueryParam("policyId") String id,
                             @QueryParam("scope") String scope,
                             @QueryParam("permission") Boolean permission,
                             @QueryParam("owner") String owner,
+                            @QueryParam("fields") String fields,
```

Comment:
@pedroigor Can this be changed to boolean? Maybe I am missing something but this string is always used only in: `fields != null && fields.equals("*")`.

Reply:
For now yes, but we may in the future use this parameter to let clients choose the fields they want back in the response.